pinot-common/src/main/java/org/apache/pinot/common/utils/helix/PinotHelixPropertyStoreZnRecordProvider.java

```diff
@@ -44,7 +44,7 @@ public static PinotHelixPropertyStoreZnRecordProvider forSchema(ZkHelixPropertyS
   }
 
   public static PinotHelixPropertyStoreZnRecordProvider forTable(ZkHelixPropertyStore<ZNRecord> propertyStore) {
-    return new PinotHelixPropertyStoreZnRecordProvider(propertyStore, "/CONFIGS/TABLES");
+    return new PinotHelixPropertyStoreZnRecordProvider(propertyStore, "/CONFIGS/TABLE");
```

Comment:
How did you find this ? DId you add a new test ?

Reply:
Or is this fly-by fix (unrelated to the changes in this PR) ?

Reply:
This was defined but not used until now. I used it and fixed it.